modules/passkey/CHANGELOG.md

```diff
@@ -38,7 +42,7 @@ Solidity compiler: [0.8.24](https://github.com/ethereum/solidity/releases/tag/v0
 Solidity optimizer: enabled with 10.000.000 runs
 EVM target: Paris
 
-## Expected addresses
+## Expected addresses with [Deterministic Deployment Proxy](https://github.com/Arachnid/deterministic-deployment-proxy)
```

Comment:
```suggestion
## Expected addresses
```

Just like above, the distinction is not at all useful here IMO (not something that I think ppl care about when reading the changelog).

Reply:
Done 👍🏾